__tests__/app.test.js

```diff
@@ -311,4 +311,45 @@ describe('app', () => {
       })
     })
   })
+
+  describe('/api/users', () => {
+    describe('GET', () => {
+      it('200: responds with an array of user objects with username, name, and avatar_url properties', () => {
+        return request(app)
+          .get('/api/users')
+          .expect(200)
+          .then(({ body }) => {
+            const { users } = body;
+
+            expect(Array.isArray(users)).toBe(true);
+
+            const expected = [
+              {
+                username: 'butter_bridge',
+                name: 'jonny',
+                avatar_url: 'https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg'
+              },
+              {
+                username: 'icellusedkars',
+                name: 'sam',
+                avatar_url: 'https://avatars2.githubusercontent.com/u/24604688?s=460&v=4'
+              },
+              {
+                username: 'rogersop',
+                name: 'paul',
+                avatar_url: 'https://avatars2.githubusercontent.com/u/24394918?s=400&v=4'
+              },
+              {
+                username: 'lurker',
+                name: 'do_nothing',
+                avatar_url: 'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png'
+              }
+            ]
+            for (let i = 0; i < users.length; i++) {
+              expect(users[i]).toMatchObject(expected[i]);
+            }
```

Comment:
You can also simplify this by just expecting users toEqual expected, since they will be the same array